source/gui/settingsDialogs.py

```diff
@@ -1135,75 +1170,104 @@ def makeSettings(self, settingsSizer):
+		
 		# Translators: This message is presented in the document formatting settings dialogue
 		# If this option is selected, NVDA will report line spacing if available. 
-		self.lineSpacingCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report &line spacing"))
+		lineSpacingText=_("&Line spacing")
+		self.lineSpacingCheckBox=pageAndSpaceGroup.addItem(wx.CheckBox(scrolledItems,label=lineSpacingText))
 		self.lineSpacingCheckBox.SetValue(config.conf["documentFormatting"]["reportLineSpacing"])
-		settingsSizer.Add(self.lineSpacingCheckBox,border=10,flag=wx.BOTTOM)
+
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		self.tablesCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report &tables"))
+		alignmentText = _("&Alignment")
+		self.alignmentCheckBox=pageAndSpaceGroup.addItem(wx.CheckBox(scrolledItems,label=alignmentText))
+		self.alignmentCheckBox.SetValue(config.conf["documentFormatting"]["reportAlignment"])
+
+		# Translators: This is the label for a group of document formatting options in the 
+		# document formatting settings dialog
+		tablesGroupText = _("Table information")
+		tablesGroup = guiHelper.BoxSizerHelper(scrolledItems, sizer=wx.StaticBoxSizer(wx.StaticBox(scrolledItems, label=tablesGroupText), wx.VERTICAL))
+		checkboxItems.addItem(tablesGroup)
+
+		# Translators: This is the label for a checkbox in the
+		# document formatting settings dialog.
+		self.tablesCheckBox=tablesGroup.addItem(wx.CheckBox(scrolledItems,label=_("&Tables")))
 		self.tablesCheckBox.SetValue(config.conf["documentFormatting"]["reportTables"])
-		settingsSizer.Add(self.tablesCheckBox,border=10,flag=wx.BOTTOM)
+
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		self.tableHeadersCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report table row/column h&eaders"))
+		self.tableHeadersCheckBox=tablesGroup.addItem(wx.CheckBox(scrolledItems,label=_("Row/column h&eaders")))
 		self.tableHeadersCheckBox.SetValue(config.conf["documentFormatting"]["reportTableHeaders"])
-		settingsSizer.Add(self.tableHeadersCheckBox,border=10,flag=wx.BOTTOM)
+
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		self.tableCellCoordsCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report table cell c&oordinates"))
+		self.tableCellCoordsCheckBox=tablesGroup.addItem(wx.CheckBox(scrolledItems,label=_("Cell c&oordinates")))
 		self.tableCellCoordsCheckBox.SetValue(config.conf["documentFormatting"]["reportTableCellCoords"])
-		settingsSizer.Add(self.tableCellCoordsCheckBox,border=10,flag=wx.BOTTOM)
+
+		# Translators: This is the label for a group of document formatting options in the 
+		# document formatting settings dialog
+		otherGroupText = _("Elements")
+		otherGroup = guiHelper.BoxSizerHelper(scrolledItems, sizer=wx.StaticBoxSizer(wx.StaticBox(scrolledItems, label=otherGroupText), wx.VERTICAL))

```

Comment:
I think this should be renamed to `elementsGroup` given the label change to `Elements`.